seed/views/data_quality.py

```diff
@@ -402,6 +402,12 @@ def save_data_quality_rules(self, request, pk=None):
         dq = DataQualityCheck.retrieve(organization.id)
         dq.remove_all_rules()
         for rule in updated_rules:
+            if rule['severity'] == Rule.SEVERITY_VALID and rule['status_label_id'] is None:
+                return JsonResponse({
+                    'status': 'error',
+                    'message': 'Label must be assigned when using Valid Data Severity.'
```

Comment:
@adrian-lara -- does this need to be translated?

Reply:
I actually haven't checked/accounted for translations of text coming from responses like this.

But yes, if the message is parsed and displayed to the user within the UI (not just in the dev tools), a translation would require a new lokalise entry.

Reply:
@adrian-lara can you add a translation for this one? Sorry to assign it to you. We can circle back with @lainsworth8801 on how it works post-release. Thanks!

Reply:
No worries! Just added it 👍